src/index.js

```diff
@@ -9,6 +9,8 @@ import store from 'store'
 
 const defaultBridgeServer = 'https://bridge.originprotocol.com'
 const defaultIpfsDomain = 'gateway.originprotocol.com'
+const defaultApolloServer = 'http://apollo.originprotocol.com'
```

Comment:
Let's use discovery (more generic) rather than Apollo (specific to framework used for implementation):
`const defaultDiscoveryServer = 'https://discovery.originprotocol.com'`

Reply:
good point